sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
```

Comment:
Nit: I would already put a release date here.

Reply:
Not sure exactly when we are going to release the package. I'll update the date right before the release.